Makefile

```diff
@@ -110,18 +120,11 @@ ci-benchmarks:
 .PHONY: ci-all-tests
 ci-all-tests: ci-benchmarks ci-integration-tests
 
-$(DEV_SSH_SERVER_KEY):
-ifeq (,$(wildcard $(DEV_SSH_SERVER_KEY)))
-	ssh-keygen -t ed25519 -N '' -f $(DEV_SSH_SERVER_KEY) < /dev/null
-endif
-
 .PHONY: start-dev
-start-dev: builder teardown-dev $(DEV_SSH_SERVER_KEY)
-	docker run -d \
+start-dev: dev-image teardown-dev
+	docker run -id \
 		--name collector_remote_dev \
-		--cap-add sys_ptrace -p127.0.0.1:$(LOCAL_SSH_PORT):22 \
-		-v $(DEV_SSH_SERVER_KEY):/etc/sshkeys/ssh_host_ed25519_key:ro \
-		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG)
+		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG)-dev
```

Comment:
We might want to mount the source directory here directly, now that we don't have ssh for mapping it, what do folks think?
```make
start-dev: dev-image teardown-dev
	docker run -id \
		--name collector_remote_dev \
		-v $(CURDIR):$(CURDIR) \
		quay.io/stackrox-io/collector-builder:$(COLLECTOR_BUILDER_TAG)-dev

Reply:
Absolutely, this will better match the VSCode "way", in particular.